extensions/wikia/Email/templates/avatarLayout.mustache

```diff
@@ -27,7 +27,10 @@
 						</a>
 					{{/ editorProfilePage}}
 				</p>
-				<p>{{details}}</p>
+				{{# detailsHeader}}
+					<p><strong>{{detailsHeader}}</strong></p>
+				{{/ detailsHeader}}
```

Comment:
I'm sure you already know this, but SOC-608 is doing essentially the same thing [here](https://github.com/Wikia/app/pull/7167/files#diff-b467f3bb9d5d158bafee32774e7a8506R32), we're going to have to resolve that conflict.